meta/validation.json

```diff
@@ -50,13 +50,11 @@
         "maxItems": { "$ref": "#/$defs/nonNegativeInteger" },
         "minItems": { "$ref": "#/$defs/nonNegativeIntegerDefault0" },
         "uniqueItems": {
-            "type": "boolean",
-            "default": false
+            "type": "boolean"
         },
         "maxContains": { "$ref": "#/$defs/nonNegativeInteger" },
         "minContains": {
-            "$ref": "#/$defs/nonNegativeInteger",
-            "default": 1
```

Comment:
this should stay

Reply:
we're okay with using "default" totally contradictory to the spec? If this is here purely for humans, it should be in a `$comment`.

Reply:
I don't think this use is contradictory to the spec, because the spec defines `default` so broadly that there is no wrong way to use it. Since there is no right or wrong way to use it and it doesn't do anything anyway, I'd rather use it in the spirit of how it was intended to be used.

> If this is here purely for humans, it should be in a `$comment`.

I had the same thought at first, but I've come to the conclusion that just because `default` is not useful as an annotation, doesn't mean it's not useful. Consider that a documentation generator doesn't use instances and therefore doesn't work on annotations as they are defined by the spec. The documentation generator can still make good use of `default` because the limitations of using it as an annotation don't apply in that context.

Reply:
I'm in agreement with this. We should consider other applications beyond just being used with an instance, the most obvious, to me, being writing schemas by hand or using UI tools (in terms of adding these for the meta schema).